index.html

```diff
@@ -18,6 +18,7 @@
 
     <script type="text/javascript">
       window.mmVersion = "#VERSION#";
+      window.intest = "#TESTMODE#";
```

Comment:
regarding naming: what does "intest" mean, reminds me of "intestines" :-) maybe just "mmTestMode" ?

Reply:
in_test   

but your suggestion is ok